.github/workflows/rust.yml

```diff
@@ -102,6 +73,34 @@ jobs:
           args:                    --manifest-path ./bin/millau/node/Cargo.toml --no-default-features --features runtime-benchmarks --verbose
 
 ## Build Stage
+  build:
+    name:                          Build
```

Comment:
AFAICT we do `--release` build, but I don't think anyone is using the binaries, so we could cut down the time a bit by doing debug build instead.